README.md

```diff
@@ -237,6 +238,20 @@ go install github.com/spf13/cobra-cli@latest
 cobra-cli add transfer
 ```
 
+### TUI Logs
+
+Because the TUI that is launched when using `dev run` manipulates the terminal,
+the logs that would usually go to `stdout` or `stderr` are written to a log
+file. You can find this log file in the `~/.astria/<instance>` directory that
+the CLI is using for `dev run`.
+For example, if you run `astria-go dev run`, the log file for the
+currently running TUI will be at `~/.astria/default/ui.log`.
+
+> NOTE: This log file gets overwritten every time you run the TUI. If you need
```

Comment:
How does this file behave if you were to, e.g.
* open terminal window and `dev run`
* open terminal window and `tail -f ~/.astria/default/astria-go.log`
* quit tui
* rerun `dev run`

Reply:
Tailing the log is unaffected by restarting the tui. The window running `tail` will show all the logs continuously across restarts/runs, but the `astria-go.log` will only show the logs for the current ui session.